frontend/src/store/config.js

```diff
@@ -206,6 +207,57 @@ export const useConfigStore = defineStore('config', () => {
     return state.value?.defaultNodesCIDR ?? '10.250.0.0/16'
   })
 
+  const shootAdminKubeconfig = computed(() => {
+    return state.value?.shootAdminKubeconfig
+  })
+
+  const allShootAdminKubeconfigExpirations = [
```

Comment:
This shoul not be defined in the configStore. The format of this list is defined by a UI component and should be dfined there.

Reply:
We could keep the allowed number values in the store but not the humanized strings. We should humanize these values in the UI component e.g
```js
function humanizeDuration (val) { 
    if (val < 3600) {
        return `${Math.floor(val/60)}m`
    }
    if (val < 86400) {
        return `${Math.floor(val/3600)}h`
    }
    return `${Math.floor(val/86400)}d`
}
```